covid19_sfbayarea/data/sonoma.py

```diff
@@ -117,24 +117,48 @@ def transform_cases(cases_tag: element.Tag) -> Dict[str, TimeSeries]:
 
     return { 'cases': cases, 'deaths': deaths }
 
-def transform_transmission(transmission_tag: element.Tag) -> Dict[str, int]:
+def transform_transmission(transmission_tag: element.Tag) -> Dict[str, float]:
     """
     Takes in a BeautifulSoup tag for the transmissions table and breaks it into
     a dictionary of type:
-    {'community': -1, 'from_contact': -1, 'travel': -1, 'unknown': -1}
+    {
+        "household": -1.0,
+        "workplace": -1.0,
+        "congregate_care": -1.0,
+        "gathering_small": -1.0,
+        "health_care": -1.0,
+        "gathering_large": -1.0,
+        "travel": -1.0,
+        "other": -1.0,
+        "unknown": -1.0
+    }
     """
     transmissions = {}
     rows = parse_table(transmission_tag)
+
     # turns the transmission categories on the page into the ones we're using
-    transmission_type_conversion = {'Community': 'community', 'Close Contact': 'from_contact', 'Travel': 'travel', 'Under Investigation': 'unknown'}
+    transmission_type_conversion = {
+        'Congregate Care': 'congregate_care',
+        'Health Care': 'health_care',
+        'Household': 'household',
+        'Large Gathering': 'gathering_small',
+        'Other': 'other',
+        'Small Gathering': 'gathering_large',
+        'Travel': 'travel',
+        'Unknown': 'unknown',
+        'Workplace': 'workplace'
+    }
+
     assert_equal_sets(transmission_type_conversion.keys(),
-                      (row['Source'] for row in rows),
+                      (row['Exposure Location'] for row in rows),
                       description='Transmission types')
+
     for row in rows:
-        type = row['Source']
-        number = parse_int(row['Cases'])
+        type = row['Exposure Location']
+        percent_cases = round(float(row['All Time'].replace('%', '')), 2)
         type = transmission_type_conversion[type]
-        transmissions[type] = number
+        transmissions[type] = percent_cases
```

Comment:
> There's an argument to be made for actually running the calculation of the percentage against the case totals, and surfacing that. Or not.

I think we should run the calculation (for consistency with other counties). I think the choice to include the more granular categories here is reasonable, but we should still keep values in the same *format/units/meaning* (counts instead of rates) to make using it at least somewhat sane.

This makes me wish we’d treated all the categorizations like age (a list of dicts rather than a dict); that way we’d have a place to put both the count and the rate. Since we didn’t we should just note in the `meta_from_baypd` that the values here are calculations from rates provided by the county, rather than exact counts.

Reply:
Also, FWIW, this is why the tests are failing.